src/Elm/Compiler/Error.elm

```diff
@@ -1,10 +1,14 @@
-module Elm.Compiler.Error exposing
-    ( Error(..), toString
-    , ParseError(..), ParseProblem(..), ParseContext(..)
-    , DesugarError(..)
-    , TypeError(..)
-    , EmitError(..)
-    )
+module Elm.Compiler.Error
+    exposing
+        ( DesugarError(..)
+        , EmitError(..)
+        , Error(..)
+        , ParseContext(..)
+        , ParseError(..)
+        , ParseProblem(..)
+        , TypeError(..)
+        , toString
+        )
```

Comment:
elm-format did this.
How much do we want to stick to elm-format?

Reply:
Let's use elm-format. We can force the multiple items on the same row using `@docs foo, bar, baz` later.